### PR TITLE
Split up tutorial in docs into separate chapters

### PR DESCRIPTION
**Pull request by bittner**
@Kidswiss and I made a significant overhaul of the K8up documentation.

- Splits up tutorial in docs into separate chapters
- Adds documentation on `Archive` object

These changes should it make easier to remove related usage instructions from cloud services, such as [APPUiO](https://appuio.ch) ([related docs](https://github.com/appuio/docs/pull/74)).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [v0.1.9] 2020-05-04
+## [v0.1.9] 2020-05-06
 ### Added
 - The operator passes the configured annotation names to Wrestic
 ### Fixed
 - KeepJobs value of sub items in Schedule are not overridden
+### Changed
```
Comment (Kidswiss) [inline CHANGELOG.md:15]: That should go into unreleased, as v0.1.9 is already released.

Comment (bittner) [reply CHANGELOG.md]: Fixed. -- Sorry it took so long!

Comment (bittner) [reply CHANGELOG.md]: Note that the latest released version of the docs is in fact [0.1.8](https://k8up.io/). Version 0.1.9 doesn't seem to have been published (automatically) with the software release.

Comment (Kidswiss) [reply CHANGELOG.md]: Yes, looks like the docs aren't build automatically. We should fix that.

**Comment by bittner** [comment]
I'll merge the changes now. It's a rather large change set, so I suggest we do related changes in a follow-up PR.

@akosma We'd need your support with a "latest" version (either built from `master`, or as an alias to the latest stable version). Can you help us with that?

**Pull request merged**